mkdocs_click/_docs.py

```diff
@@ -135,42 +135,80 @@ def _make_plain_options(ctx: click.Context) -> Iterator[str]:
     yield ""
 
 
-def _make_table_options(ctx: click.Context) -> Iterator[str]:
-    """Create the table style options description."""
+# Unicode "Vertical Line" character (U+007C), HTML-compatible.
+# "\|" (escaped pipe) would work, too, but linters don't like it in literals.
```

Comment:
Didn't know that escaping works in Python-Markdown:flushed: HTML entities would be preferred as it depends on the parser implementation though.